virt/virt-4-12-release-notes.adoc

```diff
@@ -271,9 +251,6 @@ This step is optional if you specified a value for `spec.dataVolumeTemplates`.
 
 * Windows 11 virtual machines do not boot on clusters running in link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/security_hardening/index#con_federal-information-processing-standard-fips_assembly_installing-the-system-in-fips-mode[FIPS mode]. Windows 11 requires a TPM (trusted platform module) device by default. However, the `swtpm` (software TPM emulator) package is incompatible with FIPS. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2089301[*BZ#2089301*])
 
-* The QEMU guest agent on a Fedora 35 virtual machine is blocked by SELinux and does not report data. Other Fedora versions might be affected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2028762[*BZ#2028762*])
-** As a workaround, disable SELinux on the virtual machine, run the QEMU guest agent commands, and then re-enable SELinux.
-
 //BZ-1984442 - keep in RN until it has a permanent place in docs
 * If a single node contains more than 50 images, pod scheduling might be imbalanced across nodes. This is because the list of images on a node is shortened to 50 by default. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1984442[*BZ#1984442*])
```

Comment:
This issue was removed from 4.11 as Closed - Notabug. Not sure why it still shows up here. Please delete.